.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<!-- .storybook/preview-head.html -->
```

Comment:
Interestingly, when running storybook, I get an error:

![Screenshot from 2020-11-16 15-38-31](https://user-images.githubusercontent.com/7916523/99265332-dfe09000-2821-11eb-872b-ab885f1708fc.png)

If this is not caused by your PR or it only appears on my computer, you can ignore this comment.